documentation/bibliography.tex

```diff
@@ -1,7 +1,11 @@
-\documentclass[10pt, a4paper]{basestyle}
+\documentclass[10pt, a4paper, twoside]{basestyle}
```

Comment:
I completely hate the new format.  It is way too narrow and makes the bibliography entries hard to read with multiple word breaks and many lines per entry (even ignoring the XVIIth century oddities, we have numerous entries that span 6-7 lines).

I'd be happy to keep this document one-sided.  Alternatively, organize the two-sidedness so that the body of the text keeps the same width and the `[XXX]` handles are hanging on the left.

Reply:
> organize the two-sidedness so that the body of the text keeps the same width

Agreed, and done. The wide margins from `basestyle` make sense for the other documents where they can be used for figures or review annotations, but the bibliography doesn’t have figures and it is hard to imagine we will be doing much scribbling in its margins; this is made worse by the bibliography effectively having its own margin for the labels.